### PR TITLE
Multiplayer!

### PR DESCRIPTION
**Pull request by AGoblinKing**
biz-gif

- [x] Uses prettier-eslint autoformating, let me know if you don't like any of the options
- [x] Relay pose/status over websockets
- [x] Configurable DEV=1 env flag to denote whether to allow cors on githubpreview.dev or webaverse.com

https://github.com/webaverse/app/pull/1431/ uses this

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -112,11 +113,13 @@ async function runMediasoupWorkers()
...
 async function createExpressApp()
 {
-	logger.info('creating Express app...');
+	logger.info(`Booting...${process.env.DEV ? 'DEV' : 'LIVE'}`);
 
 	expressApp = express();
 	expressApp.use(bodyParser.json());
-
+	expressApp.use(cors({
+		origin: process.env.DEV ? /\.githubpreview\.dev$/ : /\.webaverse\.com$/
```
Comment (avaer) [inline index.js:121]: Should be careful here, this might make the app undevelopable locally, which is done through things like `local.webaverse.com`

Comment (avaer) [reply index.js:121]: I'm actually fine whitelisting all domains for now.

File: package.json [hunk 2]
```diff
@@ -14,14 +14,16 @@
...
+    "cors": "^2.8.5",
     "debug": "^4.1.1",
     "express": "^4.17.1",
     "heapdump": "^0.3.15",
     "jsonwebtoken": "^8.5.1",
     "mediasoup": "^3.6.37",
     "pidusage": "^2.0.18",
     "protoo-server": "^4.0.4",
-    "sctp": "^0.0.19"
+    "sctp": "^0.0.19",
```
Comment (avaer) [inline package.json:25]: This does not seem used.

Comment (AGoblinKing) [reply package.json:25]: I'll remove that.

File: index.js (hunk 1 above)
Comment (AGoblinKing) [reply index.js:121]: Roger, I'll get that changed

**Review by avaer — COMMENTED** [review]
The rest looks good, great work! 👏

**Review by avaer — APPROVED** [review]

**Pull request merged**